exonum-java-binding/core/rust/src/runtime/cmd.rs

```diff
@@ -145,7 +145,7 @@ fn get_node_config(context: &Context) -> NodeConfig<PathBuf> {
 fn get_path_to_default_log_config() -> String {
     let log_fallback_path = {
         let mut path = executable_directory();
-        path.push("log4j-fallback.xml");
+        path.push("../log4j-fallback.xml");
```

Comment:
What is the best way to deal with this? Keep it as is supposing that our primary target is the app? And update our `qa` and `cc` scripts (that don't rely on the app) by specifying the path to config explicitly (or moving it to correct location)?

Reply:
(here and below, with lib/java) I would not make one more distinction between a packaged app and the one we get once we build the project and can run with `cargo run`.

Currently they differ in how they load the libraries: the first requires them to be in `$origin/lib/native`, the second relies on absolute paths to libjvm and rust stdlib (I wonder though how Java loads the java_bindings library in the second case). 

Maybe we can we keep the same layout for a packaged and 'normally built' application (e.g., by moving _the script_ to `bin/`)? Is there anything else we can do?